eslzArm/subscriptionTemplates/avnmConfiguration.json

```diff
@@ -0,0 +1,690 @@
+                                    },
+                                    {
+                                        "networkGroupId": "[resourceId('Microsoft.Network/networkManagers/networkGroups', 'avnm', format('avnm-ng-{0}', parameters('location')))]"
+                                    },
+                                    {
+                                        "networkGroupId": "[resourceId('Microsoft.Network/networkManagers/networkGroups', 'avnm', format('avnm-ng-{0}', parameters('locationSecondary')))]"
+                                    }
+
+                                ]
+                            }
+                        },
+                        {
+                            "condition": "[equals(parameters('enableSecondaryRegion'), 'No')]",
+                            "type": "Microsoft.Network/networkManagers/securityAdminConfigurations/ruleCollections",
+                            "apiVersion": "2024-03-01",
+                            "name": "[format('{0}/{1}/{2}', 'avnm', 'sac-alz', 'rc-ALZ')]",
+                            "dependsOn": [
+                                "[resourceId('Microsoft.Network/networkManagers/securityAdminConfigurations', 'avnm', 'sac-alz')]",
+                                "[resourceId('Microsoft.Network/networkManagers/networkGroups', 'avnm', 'avnm-ng-all')]",
+                                "[resourceId('Microsoft.Network/networkManagers/networkGroups', 'avnm', format('avnm-ng-{0}', parameters('location')))]"
+                            ],
+                            "properties": {
+                                "appliesToGroups": [
+                                    {
+                                        "networkGroupId": "[resourceId('Microsoft.Network/networkManagers/networkGroups', 'avnm', 'avnm-ng-all')]"
+                                    },
+                                    {
+                                        "networkGroupId": "[resourceId('Microsoft.Network/networkManagers/networkGroups', 'avnm', format('avnm-ng-{0}', parameters('location')))]"
+                                    }
+
+                                ]
+                            }
+                        },
+                        {
+                            "type": "Microsoft.Network/networkManagers/securityAdminConfigurations/ruleCollections/rules",
+                            "apiVersion": "2024-03-01",
+                            "name": "[format('{0}/{1}/{2}/{3}', 'avnm', 'sac-alz', 'rc-ALZ', 'DenyMgmtInbound')]",
+                            "dependsOn": [
+                                "[resourceId('Microsoft.Network/networkManagers/securityAdminConfigurations/ruleCollections', 'avnm', 'sac-alz', 'rc-ALZ')]"
+                            ],
+                            "kind": "Custom",
+                            "properties": {
+                                "description": "Deny VM Management inbound traffic",
```

Comment:
Do you want to make a mention and link to your policy that we are replicating from  AzAdvertizer, could be nice for users to get the correlation and linkage